### DISCUSSION TITLE
[way around] no screen/black screen after login - no logs (docker image on qnap NAS - container station)

### DISCUSSION THREAD

**Discussion by thesearge** [accetto/ubuntu-vnc-xfce-g3#55] [General]
I pulled your docker image (firefox g3) from docker hub and created a container on my qnap. Container creation was made by ssh and docker commands as described in user guide. noVNC or VNC connection is no problem (tested port 36901 on container or 6901 directly). However after login with provided credentials „headless“ I only receive a dark screen. I tested different machines and browsers. I also recognized that there are no logs for this container in container-station (docker app for qnap). 
![IMG_0331](https://github.com/accetto/ubuntu-vnc-xfce-g3/assets/148621007/8f925f58-992e-496d-9f06-6561a4fa9767)
![IMG_0332](https://github.com/accetto/ubuntu-vnc-xfce-g3/assets/148621007/35ba071d-3094-4dff-a321-a33e55fb7865)
Need some help.

**Comment by accetto** [comment]
I'll try to test it over weekend.

**Reply by accetto** [reply]
However, one more thing. Using `noVNC` requires a browser supporting HTML5. Maybe is that your problem?

Have you tested it over `VNC` with one of VNC viewers?

**Reply by thesearge** [reply]
Thank you for your support. I tested different modern web browsers (chrome, safari, edge) on modern equipment. Each browser should be able to handle HTML5. I also tested VNC (Tightvnc). It is always the same. I am able to connect to vnc server. I  am asked to provide credentials (headless). Connection seems to be established, but window offering remote desktop remains black. Mouse cursor is invisible in this area. I also tested light client and full client (full client screenshot looks like this ...
![Bildschirmfoto 2023-11-16 um 20 42 24](https://github.com/accetto/ubuntu-vnc-xfce-g3/assets/148621007/2cb9387f-dd7d-462c-b19b-d2a0b5c9a1be)

I think I need to check settings within docker container ... maybe missing rights on some files ...

**Reply by thesearge** [reply]
Tested today another docker installation on iMac (Docker desktop) and everything works fine. I can connect via vnc (TigerVNC) or any web browser. Therefore problem must be related to docker installation of your image on qnap container-station. Trying to perform some tests now with /tests/test-01.sh

**Reply by thesearge** [reply]
Output looks like this ...
<img width="678" alt="Bildschirmfoto 2023-11-17 um 13 01 06" src="https://github.com/accetto/ubuntu-vnc-xfce-g3/assets/148621007/3a2e6e38-26e5-41ab-9ebe-a29c013fb2e8">

**Reply by accetto** [reply]
Unfortunately I have to confirm the problem. I've got the same result.
I've created the issue #56 from this discussion. Please continue there.
Btw, the output from above looks OK.
And also thank you for reporting the problem.

**Reply by accetto** [reply]
Possible ways around are described by the issue #56.
